src/ucp/rndv/rndv.h

```diff
@@ -12,21 +12,38 @@
 #include <ucs/datastruct/ptr_map.h>
 
 
-enum ucp_rndv_rts_flags {
-    UCP_RNDV_RTS_FLAG_TAG = UCS_BIT(0),
-    UCP_RNDV_RTS_FLAG_AM  = UCS_BIT(1)
-};
+typedef enum {
+    /* RNDV operation statuses, used only by old version for TAG protocol,
+     * kept here for wire compatibility
+     */
+    UCP_RNDV_RTS_FLAG_TAG_COMPAT_VALUE_OK       = UCS_OK,
+    UCP_RNDV_RTS_FLAG_TAG_COMPAT_VALUE_CANCELED = (uint8_t)UCS_ERR_CANCELED,
+    UCP_RNDV_RTS_FLAG_TAG                       = UCS_BIT(0),
```

Comment:
why need to define this?

Reply:
do you mean why we need `UCP_RNDV_RTS_FLAG_TAG`?

Reply:
yes: a value of 0 already means it's tag. no need another one

Reply:
removed